page_analyzer/app.py

```diff
@@ -1,8 +1,115 @@
-from flask import Flask, render_template
+import os
+from typing import Any, Dict
+from urllib.parse import urlparse
+
+import psycopg2
+from dotenv import load_dotenv
+from flask import (
+    Flask,
+    flash,
+    get_flashed_messages,
+    redirect,
+    render_template,
+    request,
+    url_for,
+)
+from validators import url
+
+load_dotenv()
```

Comment:
take a look at the [`.env.example` files purpose](https://stackoverflow.com/a/52084653) - makes sense to add one